README.md

```diff
@@ -1,5 +1,10 @@
 # Andromeda
-Andromeda is a website that allows a user to perform dimensional reduction on an uploaded CSV file.
+This Andromeda package combines the data visualization tool (andromeda) with an option to fetch data from iNaturalist observations and pair them with satellite RGB and landcover data for analysis with andromeda. Andromeda uses a multi-dimensional scaling algorithm to display data in clusters on a 2-dimensional plot. It is interactive, allowing the user to choose image representations as their points and rearrange the clusters in a manner they find appropriate. This new grouping can be fed back into the algorithm to determine the weighting on provided features that most accurately describe it. Alternatively, the user can decide to increase the weight applied to a particular feature and see how that alters the clustering (they are defaulted to equal weight).
```

Comment:
```suggestion
Andromeda allows users to visualize high dimensional data in a 2-dimensional plot using weighted multidimensional scaling. Using the interactive interface users can explore the relationships within their data by adjusting the column weights and viewing the resulting clusters. Alternatively users can arrange the items into clusters and Andromeda can determine column weights.

This Andromeda includes a page to fetch data from iNaturalist observations and pair them with satellite RGB and landcover data for analysis with andromeda. 
```
This is my attempt to put the visualization part as the Hero of the app. I tried to simplify into text so that hopefully a user without Andromeda knowledge could read this and get an idea about what they could do with the app. I'm not 100% happy with my version so future edits are probably needed.

Reply:
I think this is pretty good - succinct and to the point.

FWIW, I think we should generally avoid referring to this as "_this Andromeda_" and speaking of another viz "_andromeda_" (in lower case). This will be confusing to anyyone who isn't familiar with the history (i.e., almost everyone). If we want to acknowledge and elucidate the history a bit, we can add a _History_ section to the README (and doing so could indeed be a good idea) and do so there.